bindings/matrix-sdk-ffi/src/room_list.rs

```diff
@@ -320,11 +320,17 @@ impl RoomListItem {
     fn unsubscribe(&self) {
         self.inner.unsubscribe();
     }
+}
 
+#[uniffi::export(async_runtime = "tokio")]
+impl RoomListItem {
     async fn latest_event(&self) -> Option<Arc<EventTimelineItem>> {
         self.inner.latest_event().await.map(EventTimelineItem).map(Arc::new)
     }
+}
 
+#[uniffi::export]
+impl RoomListItem {
```

Comment:
Can this block be fused with the other non-async impl block above?

Reply:
Oh, I was asking if we could put merge this `impl` block with the one that contains  `unsubscribe` above, as they're annotated the same way.